src/tasks/validate-package.js

```diff
@@ -54,6 +58,25 @@ module.exports = function validatePackage(dir, functionHandler, restApiModule) {
 						throw routeMessage + 'error template requests custom headers but does not provide defaults';
 					}
 				}
+				if (methodConfig.error && methodConfig.error.defaultCode && !validHttpCode(methodConfig.error.defaultCode)) {
```

Comment:
I don't know, I didn't really understand why it is there. That's why I asked. Is there a special reason why you think people should use that? otherwise, it's best to keep it simple, so `code` is the default, and additional errors are extras